### DISCUSSION TITLE
How I upload a file from GitHub desktop

### DISCUSSION THREAD

**Discussion by its-error505** [community/community#62392] [New to GitHub] [answered]
How I upload a file on my repository from GitHub desktop

**Comment by its-error505** [comment]
When I want to upload a file in my repository I hvae a notification regarding authentication

**Reply by Imen-Frigui** [reply, marked as answer]
I hope this is useful to you https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/authenticating-to-github-in-github-desktop
(if you could share more about the problem like screen shots or the error message)

**Reply by FarhaKousar1601** [reply]
Thank you, it's helpful @Imen-Frigui

**Comment by its-error505** [comment]
![Uploading 16908999763315966448441354621369.jpg…]()

**Comment by its-error505** [comment]
![Uploading 1690900079396239595910724181187.jpg…]()

**Comment by its-error505** [comment]
What Is enterprises address

**Reply by Imen-Frigui** [reply]
If you work along a team you could add an enterprises  address (it's optional)
GitHub Enterprise is a self-hosted version of GitHub that organizations can run on their own servers or in their private cloud environments. Each organization's GitHub Enterprise instance would have its own unique URL or address that is specific to their deployment.
Additionally, if you are a member of that organization, you can log in to GitHub Enterprise using your organization's specific URL to access the repositories and other resources.

**Comment by 06FfN3** [comment]
carsh avitor x start

**Comment by 0xRoneet** [comment]
https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/authenticating-to-github-in-github-desktop

HopeYou find this useful
